zebra-script/Cargo.toml

```diff
@@ -2,15 +2,17 @@
 name = "zebra-script"
 version = "1.0.0-beta.25"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
+description = "Zebra script verification wrapping zcashd's zcash_script library"
 license = "MIT OR Apache-2.0"
+repository = "https://github.com/ZcashFoundation/zebra"
 edition = "2021"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 zcash_script = "0.1.12"
 
-zebra-chain = { path = "../zebra-chain" }
+zebra-chain = { path = "../zebra-chain", version = "1.0.0-beta.25" }
```

Comment:
Do the dev-dependencies need versions as well?

Reply:
Looks like the dev-dependencies do not need versions